<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecorderWindow</class>
 <widget class="QMainWindow" name="RecorderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Запись аудио</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="audioRecordList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>431</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget
{

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Устройства ввода:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inputComboBox">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>10</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QComboBox" name="outputComboBox">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>43</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Устройства вывода:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: rgb(218, 218, 218);
	border-radius: 20px;
	border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="startAudioButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>1</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(218, 218, 218);
	border-radius: 24px;
	border: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons.qrc">
       <normaloff>:/IMG/IMG/AudioMsg48x48.png</normaloff>:/IMG/IMG/AudioMsg48x48.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="stopAudioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(218, 218, 218);
	border-radius: 24px;
	border: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons.qrc">
       <normaloff>:/IMG/IMG/CancelPin50x50.png</normaloff>:/IMG/IMG/CancelPin50x50.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="addAudioRecordButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>1</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(218, 218, 218);
	border-radius: 24px;
	border: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons.qrc">
       <normaloff>:/IMG/IMG/SendCommentMsgPin60x60.png</normaloff>:/IMG/IMG/SendCommentMsgPin60x60.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="durationLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>5</y>
       <width>49</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
